refactor: reorg get user information section (#595)
logto-io/docs: docs/sdk/fragments/_get-user-information.mdx

import ScopeClaimList from './_scope-claim-list.md';

{props.getIdTokenClaimsApi && props.fetchUserInfoApi &&

<>
You can use Logto API to retrieve user information:

<ul>
  <li>

>     <code>{props.getIdTokenClaimsApi}</code>: get user information from ID token.

Reviewer: getIdTokenClaims is not an API? Also might double check all the SDKs provides that method

Reviewer: Also just like sign-in sign-out, should we provide sample code as props here? 
calling getIdTokenClaimsn and getUserInfo using different SDK.

Author: Update. Only add to those docs where related apis are available.

  </li>
  <li>
    <code>{props.fetchUserInfoApi}</code>: fetch user information from the OIDC{' '}
    <a href="https://openid.net/specs/openid-connect-core-1_0.html#UserInfo">Userinfo Endpoint</a>{' '}
    provided by Logto.
  </li>
</ul>

<p>

>   It's important to note that the user information claims that can be retrieved depend on the scopes

Reviewer 2: ```suggestion
  It's important to note that the user information claims that can be retrieved depending on the scopes
```

>   used by the user during signing-in, and considering performance and data size, the ID token may

Reviewer 2: ```suggestion
  used by the user during signing-in. And considering performance and data size, the ID token may
```

>   not contain all user claims, some user claims only available in the userinfo endpoint (see the

Reviewer 2: ```suggestion
  not contain all user claims, some user claims are only available in the userinfo endpoint (see the
```

  related list below).
</p>

</>}

> Logto uses OIDC [scopes and claims conventions](https://openid.net/specs/openid-connect-core-1_0.html#Claims) to define the scopes and claims for retrieving user information from the ID token and OIDC <a href="https://openid.net/specs/openid-connect-core-1_0.html#UserInfo">Userinfo Endpoint</a>. Both of "scope" and "claim" are terms from the OAuth 2.0 and OpenID Connect (OIDC) specifications.

Reviewer 2: nit:
```suggestion
Logto uses OIDC [scopes and claims conventions](https://openid.net/specs/openid-connect-core-1_0.html#Claims) to define the scopes and claims for retrieving user information from the ID token and OIDC <a href="https://openid.net/specs/openid-connect-core-1_0.html#UserInfo">Userinfo Endpoint</a>. Both of the "scope" and the "claim" are terms from the OAuth 2.0 and OpenID Connect (OIDC) specifications.
```

In short, when you request a scope, you will get the corresponding claims in the user information. For example, if you request the `email` scope, you will get the `email` and `email_verified` data of the user.

> By default, Logto SDK will always requests three scopes: `openid`, `profile`, and `offline_access`, and there is no way to remove these default scopes. But you can add more scopes when configuring the Logto:

Reviewer 2: ```suggestion
By default, Logto SDK will always request three scopes: `openid`, `profile`, and `offline_access`, and there is no way to remove these default scopes. But you can add more scopes when configuring Logto:
```
